README.md

```diff
@@ -24,14 +24,86 @@ Supports Node v8.3+ and modern browsers.
 
 ## Usage
 
+### CLI
+
+Spectral can be run via the command-line:
+
+```bash
+spectral lint petstore.yaml
```

Comment:
As we don't instruct users to install globally, is this technically correct?

Reply:
We are adding this in https://github.com/stoplightio/spectral/pull/113